Server/Content/Sources/Run/main.swift

```diff
@@ -0,0 +1,9 @@
+import App
+import Vapor
+
+var env = try Environment.detect()
+try LoggingSystem.bootstrap(from: &env)
+let app = Application(env)
+defer { app.shutdown() }
+try configure(app)
+try app.run()
```

Comment:
Also you can clean this up a little bit, I know it one generated by default, but tbh it is quite messy. [Whitespace here and there make it more readable](https://github.com/crococoders/teztez/blob/develop/Server/Gateway/Sources/Run/main.swift)